src/components/reference/_troubleshooting-pr-links.mdx

```diff
@@ -0,0 +1,2 @@
+- [<i class="fa-regular fa-file-lines"></i> Why am I not receiving PR or MR comments?](/kb/semgrep-cloud-platform/missing-pr-commments/)
+- [<i class="fa-regular fa-file-lines"></i> Why did the comments on a PR or MR not appear inline?](/kb/semgrep-cloud-platform/inline-pr-commments/)
```

Comment:
for some reason, I can't get this link to work in the local preview

Reply:
Fixed.